cit-api/pipeline/importers/utils.py

```diff
@@ -69,17 +79,19 @@ def import_data_into_point_model(resource_type, Model, row, dry_run=False):
     import_variable_fields(instance, row, Model)
 
     instance.save()
-    calculate_distances(instance, dry_run=dry_run)
+    # calculate_distances(instance, dry_run=dry_run)
 
     return instance
 
 
 def import_data_into_area_model(resource_type, Model, row):
-    print(row)
 
     name_fields = Model.NAME_FIELD.split(",")
     name = ", ".join([str(row[name_field]) for name_field in name_fields])
 
+    if name is None:
+        name = ""
+
     instance, created = Model.objects.get_or_create(name=name)
```

Comment:
Will this cause a collision on multiple empty names?

Reply:
that I have changed in another PR as it doesnt actually do anything. I will remove it though. turns out name isnt None its "None" so that never matches.